examples/environment.yml

```diff
@@ -1,13 +1,14 @@
 name: ds
 channels:
+ - bokeh/label/dev
  - conda-forge
  - ioam/label/dev
  - ioam
  - scitools/label/dev
 
 dependencies:
  - attrs
- - bokeh
+ - bokeh=0.12.6dev3
```

Comment:
Is there a reason that this is "=", not ">="?  That's a pretty draconian restriction.